src/Form.js

```diff
@@ -82,9 +53,66 @@ const Form = (props) => {
   const PUBLIQ_GITHUBUSERCONTENT_SCHEME =
     'https://raw.githubusercontent.com/cultuurnet/apidocs/';
   const UDB_ENTRY_SCHEME_URL = `${PUBLIQ_STOPLIGHT_SCHEME}uitdatabank/nodes/reference/entry.json?deref=optimizedBundle`;
+  const UDB_SEARCH_SCHEME_URL = `${PUBLIQ_STOPLIGHT_SCHEME}uitdatabank/nodes/reference/search.json?deref=optimizedBundle`;
+  const UDB_TAXONOMY_SCHEME_URL = `${PUBLIQ_STOPLIGHT_SCHEME}uitdatabank/nodes/reference/taxonomy.json?deref=optimizedBundle`;
   const UITPAS_API_SCHEME_URL = `${PUBLIQ_STOPLIGHT_SCHEME}uitpas/nodes/reference/uitpas.json?deref=optimizedBundle`;
   const MPM_PARTNER_API_SCHEME_URL = `${PUBLIQ_STOPLIGHT_SCHEME}museumpassmusees/nodes/reference/partner-api.json?deref=optimizedBundle`;
 
+  let scheme;
+  let authMethod;
+  let defaultEnvironment = 'test';
+  let environments = [
```

Comment:
Looks good 👍 
I'm just not a big fan of all those lets and changing the values with a switch case...
Maybe it could be an idea to make a const apiTypeConfig which contains all this logic. In this case you can move it outside of the component. 

E.g. 
`const apiTypeConfig = {
 ['udb-entry']: {
 scheme: UDB_ENTRY_SCHEME_URL,
 authMethod:  'token',
 environments: [Environments.ACC, Environments.TEST, Environments.PROD]
 }
 ...
}`

Reply:
Good point 👍 See commits below

Reply:
Cool thx!